modal/functions.py

```diff
@@ -160,6 +162,8 @@ async def pop_function_call_outputs(
                 # update timeout in retry loop
                 backend_timeout = min(OUTPUTS_TIMEOUT, t0 + timeout - time.time())
                 if backend_timeout < 0:
+                    if len(response.outputs) == 0 and response.num_unfinished_inputs == 0:
+                        raise OutputExpiredError()
```

Comment:
suggestion: raising here means we change the behavior of `.remote()`. while this is arguably more consistent, maybe it would be better to keep the surface area smaller to just `.get()`?